user_data/strategies/Strategy002.py

```diff
@@ -15,9 +15,12 @@
 
 from user_data.strategies.util import execute, back_test
 from user_data.strategies.config import Config
+from user_data.strategies.notifier import send_start_deliminator_message
 
 
 class Strategy002(IStrategy):
+    if Config.IS_BACKTEST:
+        send_start_deliminator_message('Freq Strategy002 ',Config.BACKTEST_COIN ,Config.BACKTEST_MONTH_LIST[Config.BACKTEST_DATA_CLEANER_MONTH_INDEX], Config.BACKTEST_DATA_CLEANER_YEAR)
```

Comment:
do not hardcode, please refer how I got the brain name

Reply:
@tef-github tried it your way like this Freq_" + self.class.name but since it is done out side of method "self" wont work even if I create a method I still need to call the method using self so that will take me to the original problem